src/components/custom/WorkHome/WorkHome.tsx

```diff
@@ -42,8 +49,14 @@ const WorkHome = () => {
     };
 
     const saveDescription = () => {
-        setWorks([workUpdateDraft.data]);
-        setUpdateDescriptionModeOff();
+        try {
+            const temp = workUpdateDraft.data.description ?? description;
```

Comment:
```suggestion
            const temp = workUpdateDraft.data.description;
```
why are you passing existing desc into the api call?

Reply:
the assignment raised an issue of the data being a string or undefined, hence I push the existing description into temp in case. The use of workUpdateDraft.data.description is to try and get the data from the updatestore for consistency in case of any failure, it'll use description then.

Reply:
instead use `?? ""` saying that if desc is not there then show nothing